### DISCUSSION TITLE
Inline HTML rendered in Edit mode

### DISCUSSION THREAD

**Discussion by DutchPete** [notaapp/talk#418] [Feature Requests] [answered]
Things like `<span style="color:red">text</span>` are only rendered in Preview mode.
Could they be rendered in Edit mode too?

**Comment by astoilkov** [comment, marked as answer]
We've experimented with this in the past. For now, we've decided to not support this.

**Comment by DutchPete** [comment]
Thank you for a clear answer. I hope you will reconsider this sometime in the future.
